src/Microsoft.AspNetCore.OData/Results/ConflictODataResult.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+#if NETCOREAPP3_1 || NETCOREAPP2_1 || NETSTANDARD2_1
```

Comment:
why needs this? but others don't

Reply:
+1

---
In reply to: [541162024](https://github.com/OData/WebApi/pull/2341#discussion_r541162024) [](ancestors = 541162024)

Reply:
`ConflictResult`  was introduced in `AspNet core 2.1`, which is implemented from `.Net standard 2.1`
https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.controllerbase.conflict?view=aspnetcore-2.1
#Resolved